src/getScreenshots.js

```diff
@@ -39,9 +41,23 @@ const generateSnapShotPromises = (SnapShotter, config) =>
     });
     return accum;
   }, []);
+};
+
+async function executeOnBeforeSuiteScript(scriptPath) {
+  if (scriptPath) {
+    await executeScript(scriptPath).catch(error => {
+      logger.error(
+        'getSnappshots',
+        `❌  Unable to run onBeforeSuite script:\n  due to: ${error}`
```

Comment:
I think this might be swallowing the error. 
although we print it out I think takeSnapshot might still continue